functions/go/inflate-helm-chart/helm_chart_inflator_generator.go

```diff
@@ -0,0 +1,309 @@
+// Copyright 2021 The Kubernetes Authors.
```

Comment:
Are these code copied from kustomize?
If so, we should keep in `third_party` like https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/go/starlark/third_party and https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/ts/kubeval/third_party

Reply:
I moved the copied code to `third_party`